index.d.ts

```diff
@@ -131,4 +142,20 @@ export class js {
    * easystar.setIteratonsPerCalculation().
    */
   calculate(): void
+
+  /**
+   * Sets a directional condition on a tile
+   *
+   * @param {Number} x The x value of the point.
+   * @param {Number} y The y value of the point.
+   * @param {Array.<String>} list of all the allowed directions from which the tile is accessible.
+   *
+   * eg. easystar.setDirectionalCondition(1, 1, ['top']): You can only access the tile by walking down onto it,
```

Comment:
I think this comment is incorrect now that we changed `'top'` to `EasyStar.TOP`.